cypress/integration/seededFlows/tagsFlows/followTag.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Follow tag', () => {
 
     it('Follows and unfollows a tag from the tag index page', () => {
       cy.intercept('/follows').as('followsRequest');
-      cy.findAllByRole('button', { name: 'Follow' }).first().as('followButton');
+      cy.findByRole('button', { name: 'Follow tag: tag1' }).as('followButton');
```

Comment:
I love that the unique names mean we no longer need this slightly hacky `first()` call 🔥

Reply:
> Nice work - thanks for fixing those tests! ✨ Excited to see this change in the codebase 😄

Super excited to get this merged 😄  Took more than a month to complete.